pages/api/offer.ts

```diff
@@ -1,18 +1,17 @@
-import axios from "axios";
 import { NextApiRequest, NextApiResponse } from "next";
+import AppAPI from "../../api/AppAPI";
+import MockAPI from "../../api/MockAPI";
 
 export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse
 ) {
   try {
-    let offer = await (
-      await axios.get(
-        `https://bootstrap.bolt12.org/fetchinvoice/lno1pg257enxv4ezqcneype82um50ynhxgrwdajx283qfwdpl28qqmc78ymlvhmxcsywdk5wrjnj36jryg488qwlrnzyjczs/100`
-      )
-    ).data;
-    let resp = offer;
-    console.debug(resp);
+    let api: AppAPI = new MockAPI();
```

Comment:
```suggestion
    let api: AppAPI = new MockAPI();
```

this is wrong, in each action you want create a new client?